Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py

```diff
@@ -502,27 +501,27 @@ def gcloud_clusters_set_master_authorized_network_command(client: ClusterManager
         zone: Project query zone, e.g. "europe-west2-a".
         cluster: Cluster ID, e.g. "dmst-gcloud-cluster-1".
         enable: "true" or "false"
-        cidrs: Comma seprated list of CIDRs 192.160.0.0/24,10.0.0.0/24,
+        cidrs: Comma separated list of CIDRs 192.160.0.0/24,10.0.0.0/24,
 
     Returns:
         str: Human readable.
         dict: Operation entry context.
         dict: Operation raw response.
     """
     # Perform cluster update
-    update = {
+    update: ClusterUpdate = {
```

Comment:
i don't understand? it's a dict why force it's type

Reply:
done in https://github.com/demisto/content/pull/36013/commits/44fce87cd0fc072761576f2645d4bcc3fd59b0f1